api/src/main/java/com/strategyobject/substrateclient/api/pallet/cumulus_xcm/CumulusXcm.java

```diff
@@ -0,0 +1,44 @@
+package com.strategyobject.substrateclient.api.pallet.cumulus_xcm;
+
+import com.strategyobject.substrateclient.pallet.annotation.Event;
+import com.strategyobject.substrateclient.pallet.annotation.Pallet;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash256;
+import com.strategyobject.substrateclient.rpc.api.runtime.XcmError;
+import com.strategyobject.substrateclient.scale.ScaleType;
+import com.strategyobject.substrateclient.scale.annotation.Scale;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+import java.util.Optional;
+
+@Pallet("CumulusXcm")
+public interface CumulusXcm {
+
+    @Event(index = 0)
+    @Getter
+    @Setter
+    @ScaleReader
+    class InvalidFormat {
+        private Byte[] array; //Array(size 8) u8 - type 77
```

Comment:
Potentially dumb question, but is a u8 array a byte array or an int array? I know a u8 is an unsigned byte, but asking because in the existing code I've seen something like this for u8s:

@Scale(ScaleType.U8.class)
 private Integer index;

Reply:
I asked @pfrank13 about this and at least for hashes he said that we could us a byte for Hash data which was u8 and it should be fine. But since these aren't hashes I'm not sure the same holds true. I'm going to bump up the size to be safe but if anyone can find out for sure, we could save some space